src/client/phy-ue.cc

```diff
@@ -1007,7 +1007,7 @@ void PhyUe::DoEncode(int tid, size_t tag) {
                                 cfg->LdpcConfig().ExpansionFactor())));
 
   size_t bytes_per_block =
-      kEnableMac ? (ldpc_config.NumCbLen()) >> 3
+      kEnableMac ? BitsToBytes(ldpc_config.NumCbLen())
```

Comment:
I think the reason I used `>>3` was that in some settings we end up with fractional bytes for code block size and this cause mac to fail since the remaining bits of the last bytes become "don't care" by the LDPC decoder.

Reply:
```suggestion
      kEnableMac ? ((ldpc_config.NumCbLen()) >> 3)
```